### PR TITLE
Include the whole <item> in Daily RSS template

### LINKED ISSUE #182: Date of daily/dailyrss still hard formatted
Even with https://github.com/shaarli/Shaarli/commit/bec18701801cc140d760c261dd115fda1507a0dd (Define date format in templates), date title of daily.html and dailyrss.html still hard formatted into the index.php 

    $daydate = utf8_encode(strftime('%A %d, %B %Y',linkdate2timestamp($day.'_000000'))); // Full text date
    $PAGE->assign('day',utf8_encode(strftime('%A %d, %B %Y',linkdate2timestamp($day.'_000000'))));

For example, with french locale detected, the date displayed is : "lundi 16, mars 2015"

### PR DESCRIPTION
**Pull request by ArthurHoaro**
Allow custom date format and title in templates.

Also a bit of code style review.

Fixes #182

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -871,60 +874,75 @@ function showDailyRSS()
...
+    $pageaddr = escape(indexUrl());
     echo '<?xml version="1.0" encoding="UTF-8"?><rss version="2.0">';
-    echo '<channel><title>Daily - '.$GLOBALS['title'].'</title><link>'.$pageaddr.'</link>';
-    echo '<description>Daily shared links</description><language>en-en</language><copyright>'.$pageaddr.'</copyright>'."\n";
-
-    foreach($days as $day=>$linkdates) // For each day.
-    {
-        $daydate = utf8_encode(strftime('%A %d, %B %Y',linkdate2timestamp($day.'_000000'))); // Full text date
+    echo '<channel>';
+    echo '<title>Daily - '. $GLOBALS['title'];
```
Comment (virtualtam) [inline index.php]: closing `</title>` should be on this line

File: index.php [hunk 2]
```diff
@@ -871,60 +874,75 @@ function showDailyRSS()
...
+    echo '</title><link>'. $pageaddr .'</link>';
+    echo '<description>Daily shared links</description>';
+    echo '<language>en-en</language>';
+    echo '<copyright>'. $pageaddr .'</copyright>'. PHP_EOL;
+
+    // For each day.
+    foreach ($days as $day => $linkdates) {
+        $daydate = linkdate2timestamp($day.'_000000'); // Full text date
```
Comment (virtualtam) [inline index.php]: those functions should really be replaced by a DateTime object... I've opened #270 to this purpose ;-)

File: index.php (hunk 1 above)
Comment (ArthurHoaro) [inline index.php]: Thanks. :)

**Comment by virtualtam** [comment]
Neat :)

@alexisju, could you tell us if it solves #182 for template design?

**Comment by alexisju** [comment]
Yes it does ! :-) ty

**Comment by virtualtam** [comment]
Thanks, merged :)

**Pull request merged**